app/models/service_template_ansible_playbook.rb

```diff
@@ -151,4 +151,17 @@ def update_catalog_item(options, auth_user = nil)
     end
     super
   end
+
+  def destroy(auth_user = 'system')
+    config_info = options[:config_info]
+    [:provision, :retirement, :reconfigure].each do |action|
```

Comment:
For create and update, `config_info` is a passed-in option. Here for delete there is no `config_info` passed in. We don't need to follow a pattern. I would rather implement it correctly here because it is the PR all about.